frontend/src/metabase/qb/lib/actions.js

```diff
@@ -40,6 +40,11 @@ export const toUnderlyingRecords = (card: CardObject): ?CardObject => {
     }
 };
 
+export const nestThisQuery = (card: CardObject): ?CardObject => {
+    const newCard = startNewCard("query", -1337, "card__" + card.id);
```

Comment:
Good point re: magic constants. Pulled it out and gave it a name. I think once we merge the query lib in, this should be pulled into the Question api.